utils/highway-rewards-analysis/src/main.rs

```diff
@@ -0,0 +1,201 @@
+use std::{
+    collections::{BTreeMap, HashSet},
+    fs::File,
+    io::Read,
+};
+
+use clap::Parser;
+use flate2::read::GzDecoder;
+use serde::{Deserialize, Serialize};
+
+use casper_node::consensus::{
+    highway_core::{
+        finality_detector::{
+            assigned_weight_and_latest_unit, find_max_quora, round_participation,
+            RoundParticipation,
+        },
+        State,
+    },
+    protocols::common::validators,
+    utils::Validators,
+    ClContext,
+};
+use casper_types::{EraId, PublicKey, Timestamp, U512};
+
+#[derive(Parser, Debug)]
+#[command(author, version, about, long_about = None)]
+struct Args {
+    filename: String,
+    #[arg(short, long)]
+    verbose: bool,
+}
+
+/// Debug dump of era used for serialization.
+#[derive(Debug, Serialize, Deserialize)]
+pub(crate) struct EraDump {
+    /// The era that is being dumped.
```

Comment:
This duplicates era_supervisor::debug::EraDump, no?

Reply:
Right, it does. I think I started by copying code from the node and only imported the node later, but I forgot that I could import this one from the node, too :grimacing: